src/In.ProjectEKA.HipService/wwwroot/openapi.yaml

```diff
@@ -613,17 +613,17 @@ components:
     HealthInformationRequestResponse:
       type: object
       required:
-        - transactionId
```

Comment:
looks good.

Reply:
@angshu Do you want to change request body attribute from `transactionId` to `requestId`? or It's okay to keep it transactionId.